pkg/client/client.go

```diff
@@ -88,6 +89,14 @@ func (c *Client) Namespaces() NamespaceInterface {
 	return newNamespaces(c)
 }
 
+func (c *Client) PersistentVolumes() PersistentVolumeInterface {
```

Comment:
I am not super familiar with the client stuff at this point.  When ready we should get lavalamp or smarterclayton to review that part.  Likewise kubectl and jlowdermilk